WordPress/src/main/res/values-night/colors.xml

```diff
@@ -61,6 +61,9 @@
     <color name="reader_welcome_card_background">@color/background_dark</color>
     <color name="reader_welcome_card_text">@color/white</color>
 
+    <!--Reader Recommended Blogs Card -->
+    <color name="reader_recommended_blogs_card_divider">@color/white_translucent_12</color>
+
```

Comment:
We try to avoid adding separate colors for light and dark modes wherever possible. 

Currently we have

Dark mode
`<color name="reader_recommended_blogs_card_divider">@color/white_translucent_12</color>`
Light mode
`<color name="reader_recommended_blogs_card_divider">@color/black_translucent_12</color>`

These can be replaced by using `colorOnSurface` + `disabled_alpha` (.12) which will give the same output. Wondering if we can use an [already existing](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/color/on_surface_divider.xml) color here 🤔. Lmkwyt.

Reply:
ohh that's smart! We can definitely use it :)

Reply:
Now that we've reused this existing color we can remove unused
`black_translucent_12`
`white_translucent_12`

Reply:
done